spec/page_spec.rb

```diff
@@ -107,6 +107,13 @@ def foo?
         page.load { |p| p.foo? && true }
       end
 
+      it 'should allow to load html and yields itself' do
+        class Page < SitePrism::Page; end
+        page = Page.new
+        expect(page).to receive(:foo?)
```

Comment:
Can we maybe remove this line as it's covered in other expectations - leave it as blank to leave a return link before rspec